libcodechecker/analyze/host_check.py

```diff
@@ -35,3 +36,26 @@ def check_clang(compiler_bin, env):
             LOG.error(oerr)
             LOG.error('Failed to run: "' + ' '.join(clang_version_cmd) + '"')
             return False
+
+
+def has_analyzer_feature(clang_bin, feature):
+    inputFile = tempfile.NamedTemporaryFile()
```

Comment:
the inputFile should be closed or you can use it in a with statement

Reply:
Ok, I added a `with` stmt.